src/connections/allContacts.js

```diff
@@ -35,6 +39,37 @@ export const createContact = (client, attributes) =>
 
 export const updateContact = (client, contact) => client.save(contact)
 
+const isContactSynced = contact => {
+  return Object.keys(contact.cozyMetadata?.sync || {}).length > 0
+}
+
+const getGroupRelationshipLength = contact => {
+  return contact.relationships?.groups?.data.length
+}
+
+/**
+ * Delete trashed contacts without linked to an account
+ *
+ * @param {Object} client - CozyClient
+ */
+export const deleteTrashedContacts = async client => {
+  const contactContactsTrashedQuery = buildContactsTrashedQuery()
+
+  const { data: trashedContacts } = await client.query(
+    contactContactsTrashedQuery.definition(),
```

Comment:
It should generate a warning in cozy-client, the fact to use a method here, no?

Reply:
Tu peux créer une issue côté CC ? 🙏 Histoire d'avoir le même comportement sur les 2 ?

Reply:
https://github.com/cozy/cozy-client/issues/1367